### ISSUE TITLE
Semantic conventions - Resources (k8s, containers)

### ISSUE THREAD

**Issue by jpkrohling** [open-telemetry/community#1860]

**Comment by mmanciop** [comment]
Is also the [`cloud.*` semantic conventions](https://opentelemetry.io/docs/specs/semconv/resource/cloud/) in scope for this effort?

**Comment by jsuereth** [comment]
> Is also the [cloud.* semantic conventions](https://opentelemetry.io/docs/specs/semconv/resource/cloud/) in scope for this effort?

Yes.  Although, we're going to start with k8s resource stability, then likely host machine / VM.

**Comment by birojnayak** [comment]
@jsuereth is there a timeline for cloud.* ?
